tests/integrations/python/tests/test_anthropic.py

```
@@ -1773,6 +1779,1045 @@ def test_32_document_text_input(self, anthropic_client, test_config, provider, m
+                        
+                        # Check for input_json_delta (tool input)
+                        if hasattr(delta, "type") and delta.type == "input_json_delta":
+                            if hasattr(delta, "partial_json"):
+                                # Try to extract query from partial JSON
+                                try:
+                                    import json
+                                    partial = delta.partial_json
+                                    if "query" in partial:
+                                        # Accumulate JSON to parse later
+                                        pass
+                                except:
```

Ruff (0.14.11)

Do not use bare `except`

(E722)